rotkehlchen/history/events.py

```diff
@@ -119,6 +119,11 @@ def __init__(
         self.datelocaltime = db_settings.display_date_in_localtime
         self._reset_variables()
 
+    def locations_from_csv(self) -> Set[Location]:
```

Comment:
Add a docstring explaining what this does. Something like: `Returns all locations for which events exists but are not part of an exchange for which an api key is stored`

That's what I understand this function to do.